pages/api/openPR.js

```diff
@@ -152,63 +133,73 @@ function nomineeSubmission(values, sortedSubmission) {
     organizations: values.organizations ? [values.organizations] : [],
     stage: values.stage ? values.stage : "",
   };
-
   // Order nominee fields in the correct order e.g organizations
   sortedSubmission = orderFields(sortedSubmission);
   return sortedSubmission;
 }
 
+function submitPullRequests(projectName, filesObject) {
+  const MyOctokit = Octokit.plugin(createPullRequest);
+  const octokit = new MyOctokit({
+    auth: TOKEN,
+  });
+
+  // initialize an array for Promises
+  let promiseArray = [];
+
+  for (const submission in filesObject) {
+    let singleFile = {};
+    singleFile[submission.toString()] = filesObject[submission.toString()];
+
+    // Returns a normal Octokit PR response
+    // See https://octokit.github.io/rest.js/#octokit-routes-pulls-create
+    promiseArray.push(
+      composeCreatePullRequest(octokit, {
+        owner: GITHUB_OWNER,
+        repo: GITHUB_REPO,
+        title: `Add nominee: ${projectName}`,
```

Comment:
How does `nominee` change to `DPG` if it's hardcoded here? It should be dynamic, shouldn't it? I am confused by looking at the tests [here](https://github.com/nathanfletcher/submission-digitalpublicgoods/pull/3) and [here](https://github.com/nathanfletcher/submission-digitalpublicgoods/pull/4), and seeing different text.

Reply:
I see now that this has been addressed in 46e1f2d, I was looking at an outdated version of the code.

Reply:
Yes, you're correct @nathanbaleeta pointed it out and I made a fix in a recent commit. But I'll do a double check.